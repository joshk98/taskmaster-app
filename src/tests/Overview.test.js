import React from "react";
import { render } from "@testing-library/react";
import Overview from "../components/Overview";

describe("Overview", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Overview />);

    expect(asFragment()).toMatchSnapshot();
  });
});
