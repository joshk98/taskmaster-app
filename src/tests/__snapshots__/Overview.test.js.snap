// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview renders correctly 1`] = `
<DocumentFragment>
  <div>
    Overview
  </div>
</DocumentFragment>
`;
